File: Assignment1/7.c
#include <stdio.h>
#include <limits.h>

int main() {
    int n;
    printf("Enter size of the array: ");
    scanf("%d", &n);

    int arr[n];
    printf("Enter %d elements:\n", n);
    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    int maxi = INT_MIN;
    int mini = INT_MAX;

    for (int i = 0; i < n; i++) {
        if (arr[i] > maxi)
            maxi = arr[i];
        if (arr[i] < mini)
            mini = arr[i];
    }

    printf("Maximum value in array: %d\n", maxi);
    printf("Minimum value in array: %d\n", mini);

    return 0;
}

File: Week-1/7.py
n = int(input("Enter size of the array: "))
arr = []
print(f"Enter {n} elements: ")
for i in range(n):
    arr.append(int(input()))

maxi = arr[0]
mini = arr[0]

for x in arr:
    if x > maxi:
        maxi = x
    if x < mini:
        mini = x

print("Maximum value in array: ",maxi)
print("Minimum value in array: ",mini)